migration/migrations/20200707091140-add-transfer_group-to-order.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    queryInterface.addColumn(
+      'Orders',
+      'transfer_group',
+      Sequelize.STRING
+    );
+  },
+
+  down: (queryInterface, Sequelize) => {
+    queryInterface.removeColumn(
+      'Orders',
+      'transfer_group'
+    );
+  }
+};
```

Comment:
@namandangi  This is not related to this PR, I don't know why these changes are here